plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchMetadata.java

```diff
@@ -579,16 +585,87 @@ public ConnectorTableProperties getTableProperties(ConnectorSession session, Con
         return Optional.of(new ConstraintApplicationResult<>(handle, TupleDomain.withColumnDomains(unsupported), newExpression, false));
     }
 
-    protected static String likeToRegexp(String like)
+    protected static boolean checkArgument(Call call, int index, Predicate<ConnectorExpression> predicate)
```

Comment:
`checkCallArgument`...  but it's still too similar to `checkArgument` which throws when condition is not satisfied.

so inline